src/widgets/speakers/ui/speakers.tsx

```diff
@@ -21,15 +26,19 @@ export const Speakers = () => (
           So don&apos;t hesitate to drop a short synopsis to RS Head
         </Paragraph>
 
-        <div className="name" data-testid="contact-name">
+        <Subtitle color="black" className={cx('name')}>
           Dzmitry Varabei
-        </div>
-        <div className="email">
-          <EmailIcon />
-          <span>rolling.scopes@gmail.com</span>
-        </div>
-      </div>
-      <Image className="right picture" src={image} alt="speakers-wanted" />
+        </Subtitle>
+        <address className={cx('email-wrapper')}>
+          <Image src={email} alt="" aria-hidden="true" />
+          <span className={cx('email')}>{RS_EMAIL}</span>
```

Comment:
Should we consider using `a href="mailto` here?

Reply:
should it be styled as link in this case?

Reply:
I'd say yes, since the link should look like a link

Reply:
done